modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc

```diff
@@ -63,7 +63,7 @@ $ sudo subscription-manager repos --enable=rhel-9-for-<architecture>-appstream-r
 +
 [NOTE]
 ====
-For more information about Red Hat Subscription Manager, see link:https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html-single/rhsm/index[Using and Configuring Red Hat Subscription Manager].
+For more information about Red Hat Subscription Manager, see link:https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_rhel_system_registration/basic-reg-rhel-cli[Registering a RHEL system with command line tools].
```

Comment:
By right, `command-line tools` fits the examples in the IBM SG and RH SG. The follow-up link drops the hyphen and that is outside your remit to fix on the RHEL doc. I do not see an issue with fixing the missing hyphen in the link metadata for both cases in this PR.

An attribute exists for RHEL, consider using the attribute:

![Screenshot from 2024-11-27 14-55-31](https://github.com/user-attachments/assets/48875ce4-e367-4d69-8d78-9a480ce9799b)

Reply:
Updated as suggested.